action.yml

```diff
@@ -1,78 +1,79 @@
+          elif [ ${{ github.event.discussion }} ]; then
+            type=discussion_description
+            content=$DISCUSSION_BODY
+            discussion_node_id='${{ github.event.discussion.node_id }}'
+            user=${{ github.event.discussion.user.login }}
+            target=" your discussion body"
           fi
-          message="Uh oh! @$user, the image you shared is missing helpful alt text. Check $target.
+        fi
+        npm ci
+        flag=$(node index.js "$content") 
+        message="Uh oh! @$user, the image you shared is missing helpful alt text. Check $target.
 
-          Alt text is an invisible description that helps screen readers describe images to blind or low-vision users. If you are using markdown to display images, add your alt text inside the brackets of the markdown image.
-          
-          Learn more about alt text at [Basic writing and formatting syntax: images on GitHub Docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images)."
-          flag="$(flagAltText "$content")"
-          
-          echo "Detected bad alt text: ${flag}"
-          echo "Event type: $type"
+        Alt text is an invisible description that helps screen readers describe images to blind or low-vision users. If you are using markdown to display images, add your alt text inside the brackets of the markdown image.
 
-          if [[ $flag = true ]]; then
-            if [[ $type = pr_comment ]] || [[ $type = pr_description ]]; then
-              gh pr comment $issue_url --body "$message"
-            elif [[ $type = issue_comment ]] || [[ $type = issue_description ]]; then
-              gh issue comment $issue_url --body "$message"
-            elif [[ $type = discussion_description ]]; then
-              addDiscussionComment $discussion_node_id "$message"
-            elif [[ $type = discussion_comment ]]; then
-              addDiscussionComment $discussion_node_id "$message" $reply_to_id 
-            fi
+        Learn more about alt text at [Basic writing and formatting syntax: images on GitHub Docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images)."
+
+        echo "Detected bad alt text: ${flag}"
+        echo "Event type: $type"
+
+        if [[ $flag = true ]]; then
+          if [[ $type = pr_comment ]] || [[ $type = pr_description ]]; then
+            gh pr comment $issue_url --body "$message"
+          elif [[ $type = issue_comment ]] || [[ $type = issue_description ]]; then
+            gh issue comment $issue_url --body "$message"
+          elif [[ $type = discussion_description ]]; then
+            addDiscussionComment $discussion_node_id "$message"
+          elif [[ $type = discussion_comment ]]; then
+            addDiscussionComment $discussion_node_id "$message" $reply_to_id 
```

Comment:
I would love if we can eventually move all of this bash logic into a well-tested node script! Out of scope here though :)

Reply:
Agreed!!